src/lib/system-info.ts

```diff
@@ -0,0 +1,114 @@
+import * as systeminformation from 'systeminformation';
+import * as osUtils from 'os-utils';
+import { fs, child_process } from 'mz';
+
+export function getCpuUsage() {
+	return new Promise((resolve) => {
+		osUtils.cpuUsage((percent) => {
```

Comment:
I think you might be able to replace this with `systeminformation.currentLoad` and have just a single module

Reply:
load and cpu usage is different @Page-

Reply:
@CameronDiver I know "load average" is different but the docs for that call definitely imply it's cpu load/usage:
![image](https://user-images.githubusercontent.com/360115/92123350-704b2f00-edf4-11ea-9c1a-5630dd302ebc.png)